crates/uv-cli/src/lib.rs

```diff
@@ -4288,6 +4297,10 @@ pub struct PythonListArgs {
     /// By default, these display as `<download available>`.
     #[arg(long)]
     pub show_urls: bool,
+
+    /// Select the output format between: `text` (default) or `json`.
```

Comment:
The default is already rendered in the help, I think. You may want to omit that here and just say "Select the output format".

Reply:
good catch!